packages/runtime/test/src/runtime.ts

```diff
@@ -48,26 +48,36 @@ describe("Logic Signature Transaction in Runtime", function () {
   });
 
   it("should not verify signature because alice sent it", () => {
-    txnParam.fromAccount = alice.account;
+    const invalidParams: ExecParams = {
+      ...txnParam,
+      sign: SignType.LogicSignature,
+      fromAccountAddr: alice.account.addr,
+      lsig: lsig
+    };
+    // txnParam.fromAccount = alice.account;
```

Comment:
please remove this comment if not needed

Reply:
ok